.pytest.ini

```diff
@@ -1,5 +0,0 @@
-[pytest]
-DJANGO_SETTINGS_MODULE = webtas.settings
-; python_files = tests.py test_*.py *_tests.py
-
-addopts = --cov=. --cov-config=.coveragerc --cov-report xml:cov.xml
```

Comment:
I believe this is required for the code coverage vscode extension. If we don't want it checked in, could you .gitignore it as well?

Reply:
Not aware of that extension @Kyle0654 but the `.ini` options were originally duplicated (and slightly different) in both the `pytest.ini` and `pyproject.toml` files.

https://github.com/invoke-ai/InvokeAI/blob/8f21201c91520622307d51b3875d5160672ceaaf/.pytest.ini#L5

https://github.com/invoke-ai/InvokeAI/blob/8f21201c91520622307d51b3875d5160672ceaaf/pyproject.toml#L142

I'd attempted to merge these into `pyproject.toml` which, per my understanding, is the preferred method nowadays for this sort of stuff.

What's required bit for the vscode extension? That `DJANGO_SETTINGS_MODULE` line?

Reply:
Have added to .gitignore now as requested